sources/US/MI/state-areas-2017.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name" : "MI State Forest, Wildlife, and Game Areas Open to Hunting 2017",
+    "url": "http://gis-midnr.opendata.arcgis.com/datasets/403b88ca6cc443a59e54eb28e4f4de98_5#",
```

Comment:
This url goes to a webpage, not a zip file

Reply:
Ok. the Geojson link would never load in qgis for me so I grabbed the shp file link, but it had some text after the .zip extension. 

I’ll upload the shp to The S3 bucket and pull a url from there